web4/treasury-web4/src/lib.rs

```diff
@@ -18,10 +18,11 @@ impl Contract {
         &mut self,
         widget_reference_account_id: String,
         social_db_account_id: String,
+        set_social_metadata_defaults: Option<bool>,
```

Comment:
Do I understand it right, that we will have to also update the UI components to set `set_social_metadata_defaults` to true? Why don't make it true by default?

Reply:
I would not make this true by default, since it can be used to update from reference widgets later. If you also have set your own metadata, you don't want that to be overwritten.